doc/manipulating_visualizing/manipulating_images.rst

```diff
@@ -222,14 +234,15 @@ can be easily extracted from the fMRI data using the
 
 .. _mask_4d_2_3d:
 
-From 4D to 2D arrays
---------------------
+From 4D nifti images to 2D data arrays
+--------------------------------------
 
 fMRI data is usually represented as a 4D block of data: 3 spatial
-dimensions and one of time. In practice, we are most often only
-interested in working only on the time-series of the voxels in the
-brain. It is thus convenient to apply a brain mask and go from a 4D
-array to a 2D array, `voxel` **x** `time`, as depicted below:
+dimensions and one time dimension. In practice, we are usually
+interested in working on the voxel time-series in the
+brain. It is thus convenient to apply a brain mask in order to translate the
```

Comment:
I wanted to explicitly not use Transform() to paraphrase the function name intentionally.